Backend/PluginActions.php

```diff
@@ -126,8 +61,6 @@ public function patchPluginSettingsDefaultSettings() {
         $this->clientAPI->callAPI(new \CF\API\Request('PATCH', 'zones/'. $zoneId .'/settings/mirage', array(), array('value' => 'off')));
 
         $this->clientAPI->callAPI(new \CF\API\Request('PATCH', 'zones/'. $zoneId .'/settings/rocket_loader', array(), array('value' => 'off')));
-
-        return $this->api->createAPISuccessResponse($this->api->createPluginResult(\CF\API\Plugin::SETTING_DEFAULT_SETTINGS, true, true, ''));
```

Comment:
The response is returned in cloudflare/cloudflare-plugin-backend CF\API\AbstractPluginsActions.php now.